src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -49,25 +53,27 @@ private Network getNetwork(UUID networkUuid, PreloadingStrategy strategy, String
         }
     }
 
-    private static VoltageLevelMapData toMapData(VoltageLevel voltageLevel, boolean includeDetails) {
+    private static VoltageLevelMapData toMapData(VoltageLevel voltageLevel) {
         VoltageLevelMapData.VoltageLevelMapDataBuilder builder = VoltageLevelMapData.builder()
                 .id(voltageLevel.getId())
                 .topologyKind(voltageLevel.getTopologyKind());
 
-        if (includeDetails) {
-            builder.name(voltageLevel.getOptionalName().orElse(null))
-                    .substationId(voltageLevel.getSubstation().map(Substation::getId).orElse(null))
-                    .substationName(voltageLevel.getSubstation().map(Substation::getName).orElse(null))
-                    .nominalVoltage(voltageLevel.getNominalV())
-                    .lowVoltageLimit(Double.isNaN(voltageLevel.getLowVoltageLimit()) ? null : voltageLevel.getLowVoltageLimit())
-                    .highVoltageLimit(Double.isNaN(voltageLevel.getHighVoltageLimit()) ? null : voltageLevel.getHighVoltageLimit());
+        builder.name(voltageLevel.getOptionalName().orElse(null))
+                .substationId(voltageLevel.getSubstation().map(Substation::getId).orElse(null))
+                .substationName(voltageLevel.getSubstation().map(Substation::getName).orElse(null))
```

Comment:
Identifiable getName is deprecated, use getOptionalName or getNameOrId

Reply:
Done